scripts/cangen/cangen/traffic.py

```diff
@@ -0,0 +1,48 @@
+"""
+Author: Manush Patel
+Date: 2024-11-24
+https://macformula.github.io/racecar/firmware/can-traffic/
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass
+class CANMessage:
+    data_length: int
+    frequency: int
+
+    @property
+    def bit_count(self):
+        fixed_length = 44  # fixed length of CAN frame
+        variable_length = self.data_length * 8.0  # accounted for bit stuffing
```

Comment:
this comment is incorrect since this calculation does not include stuffing. Notice how really simple comment can easily become outdated.